registry/api/v2/descriptors.go

```diff
@@ -523,7 +523,7 @@ var routeDescriptors = []RouteDescriptor{
 						Successes: []ResponseDescriptor{
 							{
 								Description: "The manifest has been accepted by the registry and is stored under the specified `name` and `tag`.",
```

Comment:
I would think that "accepted" is still correct, because the client is uploading the manifest, not having the server create one.